test/mixed_workload_test.go

```diff
@@ -0,0 +1,108 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	. "github.com/onsi/gomega"
+	clientv3 "go.etcd.io/etcd/client/v3"
+)
+
+// TestMixed puts heavy load on kine and expects that some requests are denied
+// by the admission control.
+func TestMixed(t *testing.T) {
```

Comment:
Let's then maybe call this `TestAdmissionControl`?

Reply:
Done